EIPS/eip-233.md

```diff
@@ -18,13 +18,89 @@ Today discussions about hard forks happen at various forums and sometimes in ad-
 
 ## Specification
 
-A Meta EIP should be created and merged as a *Draft* as soon as a new hard fork is planned. This EIP should contain:
+A Meta EIP should be created and merged as a *Draft* as soon as a new hard fork is planned. The hard fork should be 6 - 9 months after the last hard fork.
```

Comment:
Given that there is talk about setting a proper HF schedule in the Berlin meetings in April, does it make sense to remove this line for now and re-add it when there is consensus on the HF lengths? -> `The hard fork should be 6 - 9 months after the last hard fork.`

Reply:
I'd agree it may be premature to add this line. Timelines and scheduling would deserve its own chapter.